samples/apps/smallbank/smallbank.cmake

```diff
@@ -21,6 +21,7 @@ if(BUILD_TESTS)
     CLIENT_BIN ./small_bank_client
     VERIFICATION_FILE ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank.json
     ADDITIONAL_ARGS
+      --label Small_Bank_ClientCpp
```

Comment:
We currently have this small bank test recorded under a different name on Blob Storage so I am reverting its label back to what it used to be.